sroka/api/google_ad_manager/gam_api.py

```diff
@@ -92,3 +92,65 @@ def get_data_from_admanager(query, dimensions, columns, start_date, end_date, cu
     with gzip.open(report_file.name) as file:
         data = pd.read_csv(file)
     return data
+
+
+def get_users_from_admanager(query, dimensions, network_code=None):
+    
+    user_df = pd.DataFrame()
+    
+    dimensions_df = pd.DataFrame()
+        
+    #.Where() handles filtering with admanager method, while statement_query handles queries with WHERE clause
+    
+    statement_query = query.replace("WHERE ", "")
+    
+    statement = (ad_manager.StatementBuilder()
+               .Where((statement_query)))
+    
+    if not network_code:
+        try:
+            network_code = config.get_value('google_ad_manager', 'network_code')
+        except (KeyError, NoOptionError):
+            print('No network code was provided')
+            return pd.DataFrame([])
+
+    yaml_string = "ad_manager: " + "\n" + \
+        "  application_name: " + APPLICATION_NAME + "\n" + \
+        "  network_code: " + str(network_code) + "\n" + \
+        "  path_to_private_key_file: " + KEY_FILE + "\n"
+
+    # Initialize the GAM client.
+    gam_client = ad_manager.AdManagerClient.LoadFromString(yaml_string)
+    
+    user_service = gam_client.GetService('UserService')
+    
+    
+    try:
+
+        while True:
+            response = user_service.getUsersByStatement(statement.ToStatement())
+            if 'results' in response and len(response['results']):
+              for user in response['results']:
```

Comment:
use consistent number of spaces throughout the file.

Reply:
Done!